src/main/webapp/index.jsp

```diff
@@ -1,5 +1,19 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+ 
+pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
 <html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Spring 4 MVC - HelloWorld Index Page</title>
+</head>
 <body>
-<h2>Welcome To PayDesk!</h2>
+ 
+	<center>
+		<h2>Hello World</h2>
+		<h3>
+			<a href="hello?name=Prassanna">Click Here</a>

```

Comment:
I think it would be good if we move all these dumpy pages to a separate place ? Basically when we start adding login we need to remove all these .